pyseir/deployment/demographic_mapper.py

```diff
@@ -0,0 +1,592 @@
+        - 'D': np.array, death during hospitalization
+        - 'R': recovered
+    predictions_by_age: dict
+        Contains time series of SEIR model compartment size by age groups.
+        Time series are simulated with MLE parameters.
+        With keys:
+        - 'E': np.array, exposed by age groups
+        - 'I': np.array, infected and symptomatic by age groups
+        - 'A': np.array, infected and asymptomatic by age groups
+        - 'HGen': np.array, admitted to non-ICU by age groups
+        - 'HICU': np.array, admitted to ICU by age groups
+        - 'HVent': np.array, admitted to ICU with ventilator by age groups
+    parameters: dict
+        Contains MLE model parameters. For full list of parameters,
+        refer description of parameters of SEIRModelAge.
+    measures: list(CovidMeasure)
+        Covid measures.
+    measure_units: list(CovidMeasureUnit)
+        Units of covid measure.
+    hospitalization_rates: dict
+        Rates of hospitalization by age group, type of hospitalization,
+        and unit of rates, with unit as primary key, type of
+        hospitalization as secondary key, and array of corresponding
+        time series of rates as values.
+        For example, hospitalization_rates['per_capita']['HGen'] is the
+        time series of probability of being admitted to non-ICU per
+        capita among infected population (asymptomatic + symptomatic).
+    mortality_rates: dict
+        Rates of mortality by age group, type of hospitalization,
+        and unit of rates, with unit as primary key, type of
+        hospitalization as secondary key, and array of corresponding
+        time series of rates as values.
+        For example, mortality_rates['per_capita']['HICU'] is the time
+        series of probability of death in ICU per capita among infected
+        population (asymptomatic + symptomatic + hospitalized).
+    prevalence: np.array
+        Age-specific prevalence time series simulated with SEIR model
+        with MLE parameters.
+    results: dict
+        Contains time series of covid measures predicted using the MLE model
+        and adjusted by target age distribution and risk modification,
+        with name of covid measures as primary key and name of measure unit
+        as secondary key.
+
+
+    Parameters
+    ----------
+    fips: str
+        State of county FIPS code
+    mle_model: SEIRModelAge
+        Model with age structure and MLE model parameters
+    fit_results: dict
+        MLE epi parameters and associated errors. The parameter used is
+        t0_date.
+    measures: str or list(str)
+        Names of covid measures, should be interpretable by CovidMeasure.
+    measure_units: str or list(str)
+        Units of covid measures, should be interpretable by CovidMeasureUnit.
+    target_age_distribution: callable
+        PDF of age of target population.
+    risk_modifier_by_age: callable
+        Returns risk rations by age group that modifies the risk of
```

Comment:
```suggestion
        Returns risk ratios by age group that modifies the risk of
```

Reply:
I'm not sure I really understand what the inputs and outputs of this distribution are.

Reply:
The input is an array of age and output is the pdf based on target age distribution. I have updated the description and the variable's name.